### DISCUSSION TITLE
如何在richedit中添加图片等？

### DISCUSSION THREAD

**Discussion by chenxuanw** [netease-im/NIM_Duilib_Framework#472] [General]
想要在richedit中添加图片或者照片，像qq微信一样，可以在添加图片后继续加文字，或者把图片删掉？

**Comment by nmgwddj** [comment]
参考 duilib 编写的 NIM demo，包含完整的 IM 即时通讯示例，其中包含发送图片等能力，editor 显示图片通过 image_old.dll 实现。

**Reply by chenxuanw** [reply]
例子有点庞大，是个新手，看起来比较吃力。能否给出具体怎么实现，或者相关代码的cpp名称以及大致逻辑

**Reply by nmgwddj** [reply]
https://github.com/netease-kit/NIM_PC_Demo/tree/master/tool_kits/ui_component/image_ole
这里是将本地图片加载到 RichEdit 的实现代码，可作为参考。
